### PR TITLE
chore(example): upgrade the example projects react-native version to …

### PR DESCRIPTION
**Pull request by jspizziri**
…0.71.8

Fixes https://github.com/doublesymmetry/react-native-track-player/issues/2009

### CODE REVIEW

**Comment by jspizziri** [comment]
@bradleyflood would you be able to give me a review on this? I can't merge until there's an approval.

**Review by bradleyflood — APPROVED** [review]

**Comment by bradleyflood** [comment]
Done :) Was going to review yesterday but I don’t have write so it doesn’t usually count.

**Pull request merged**
